bitrise.yml

```diff
@@ -257,47 +256,3 @@ workflows:
             #!/bin/bash
             set -ex
             stepman audit --step-yml ./step.yml
-
```

Comment:
create-release workflow not needed

Reply:
Yes, it is:
https://github.com/bitrise-io/go-utils/commit/6eb582e08179a025ed2deb86ea6dffbd13ba1ff6

Go packages don't have releases this is why it's locked on a commit hash.

Reply:
Switch godeps update with dep update workflow